warning-synonyms don't work for faceting (#1186)
meilisearch/documentation: reference/features/filtering_and_faceted_search.md

Once they have been configured, you can search for facets with the `filter` search parameter.

:::warning

> Please note that **synonyms don't work for facets.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**

Reviewer: ```suggestion
Please note that **synonyms don't apply to facets.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**
```

> Please note that **synonyms don't apply to facets.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**

Reviewer 2: Since facets are just a subset of filters and that facets are harder to explain, what do you think about this:

```suggestion
Please note that **synonyms don't apply to filters.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**
```
